docs/standard/architecture/toc.md

```diff
@@ -0,0 +1,77 @@
+### [Microservices addressability and the service registry](architecting-container-and-microservice-based-applications/microservices-addressability-and-the-service-registry.md)
+### [Creating composite UI based on microservices, including visual UI shape and layout generated by multiple microservices](architecting-container-and-microservice-based-applications/creating-composite-ui-based-on-microservices,-including-visual-ui-shape-and-layout-generated-by-multiple-microservices.md)
+### [Resiliency and high availability in microservices](architecting-container-and-microservice-based-applications/resiliency-and-high-availability-in-microservices.md)
+### [Orchestrating microservices and multi-container applications for high scalability and availability](architecting-container-and-microservice-based-applications/orchestrating-microservices-and-multi-container-applications-for-high-scalability-and-availability.md)
+### [Using Azure Service Fabric](architecting-container-and-microservice-based-applications/using-azure-service-fabric.md)
+## [Development Process for Docker Based Applications](development-process-for-docker-based-applications/)
+### [Development workflow for Docker apps](development-process-for-docker-based-applications/development-workflow-for-docker-apps.md)
+## [Deploying Single Container Based NET Core Web Applications on Linux or Windows Nano Server Hosts](deploying-single-container-based-net-core-web-applications-on-linux-or-windows-nano-server-hosts/index.md)
+## [Migrating Legacy Monolithic NET Framework Applications to Windows Containers](migrating-legacy-monolithic-net-framework-applications-to-windows-containers/index.md)
+## [Designing and Developing Multi Container and Microservice Based NET Applications](designing-and-developing-multi-container-and-microservice-based-net-applications/index.md)
+### [Designing a microservice-oriented application](designing-and-developing-multi-container-and-microservice-based-net-applications/designing-a-microservice-oriented-application.md)
+### [Creating a simple data-driven CRUD microservice](designing-and-developing-multi-container-and-microservice-based-net-applications/creating-a-simple-data-driven-crud-microservice.md)
+### [Defining your multi-container application with docker-compose.yml ](designing-and-developing-multi-container-and-microservice-based-net-applications/defining-your-multi-container-application-with-docker-composeyml-.md)
+### [Using a database server running as a container](designing-and-developing-multi-container-and-microservice-based-net-applications/using-a-database-server-running-as-a-container.md)
+### [Implementing event-based communication between microservices (integration events)](designing-and-developing-multi-container-and-microservice-based-net-applications/implementing-event-based-communication-between-microservices-(integration-events).md)
```

Comment:
I don't think file names should have parentheses plus this doesn't sync with the toc